src/modules/display.js

```diff
@@ -39,10 +40,11 @@ const getPokemon = async (res) => {
         <img src="${pokemon.sprites.other.dream_world.front_default}" alt="${pokemon.name} image">
         <div class="pokemon-title"> <h3>${pokemon.name}</h3> <i class="fa-regular fa-heart likeBtn" id="${pokemon.id}"></i></i>
         </div>
-            <div class="likes-contain"><h4 class="itemLikes">${likeCount}</h4><p>likes</p></div>
-        <div>
-          <button type="button" class="btn">Comment</button>
-        </div>
+
+        <div class="btn">
+          <button type="button" class="comment-btn">Comment</button>
+            <div class="likes-contain"><h4 class="itemLikes">${likeCount}</h4><p>likes</p>
+            </div>
```

Comment:
here as I told you yesterday kindly consider removing the class btn from the div and add it beside the class comment-btn